### DISCUSSION TITLE
Provide management script

### DISCUSSION THREAD

**Discussion by 766F6964** [dani-garcia/vaultwarden#2082] [Ideas]
Hello everyone,

I am currently in the process of trying out vaultwarden. During the setup, I immediately noticed one thing: Its very tedious to set everything up.
Bitwarden provides a little script that makes it easy to manage the bitwarden instance (install, uninstall, update, stop, start, renew cert etc). See: https://bitwarden.com/help/article/install-on-premise-linux/
I would love to have something like this for vaultwarden as well, - it would make using vaultwarden so much more convenient.

**Comment by BlackDex** [comment]
Well there isn't much to manage.
https://github.com/dani-garcia/vaultwarden/wiki/Using-Docker-Compose

You only need to use docker-compose [up, down, pull, restart] etc....

**Reply by 766F6964** [reply]
I wasn't aware that there is a docker-compose configuration that does essentially everything with one click, that is nice.
Will this automatically renew the certificate as well ?

**Reply by BlackDex** [reply]
I'm not sure, but i thought caddy for example should be able to do that yes.

**Reply by jjlin** [reply]
Yes, that page provides a Caddy config that should automate renewal.
